AspNetCoreWeb/Infrastructure/SearchOptionsProcessor{T,TEntity}.cs

```diff
@@ -51,12 +50,11 @@ private static IEnumerable<SearchTerm> GetValidTerms(IEnumerable<DTColumn> colum
                 yield break;
             }
 
-            var declaredTerms = GetTermsFromModel(typeof(T));
+            var declaredTerms = GetTermsFromModel(typeof(T)).ToList();
```

Comment:
Please remove `ToList()` we can add that if we are enumerating this more than once.

Reply:
Calling SingleOrDefault does iterate the entire list right? Or am I mistaken?

Reply:
I'm sorry. You're right. I misunderstood.